pom.xml

```diff
@@ -44,28 +44,6 @@
         <air.java.version>17</air.java.version>
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
-
-        <!-- dependency versions -->
```

Comment:
Some of these were needed as overrides since there were different versions coming in between dropwizard and airbase .. and now also airlift

Reply:
And we might have to check that our override works and potentially enforce via depMgt

Reply:
Airlift comes with latest :)